lib/XML/LibXML/CStructs.pm

```diff
@@ -113,6 +119,27 @@ my class xmlError is export(:types) {
     has OpaquePointer $.node; # the node in the tree
 }
 
+my class xmlElement is export(:types) {
```

Comment:
OK. xmlElement is now passing, but xmlSAXHandler and xmlXPathContext didn't. Do you want a separate PR for these?